crates/core_simd/src/swizzle.rs

```diff
@@ -0,0 +1,374 @@
+use crate::simd::intrinsics;
+use crate::{LaneCount, Simd, SimdElement, SupportedLaneCount};
+
+/// Rearrange vector elements.
+///
+/// A new vector is constructed by specifying the the lanes of the source vector or vectors to use.
```

Comment:
This should prefer active voice and possibly less repetition, e.g.
```suggestion
/// Constructs a new vector by selecting values from the lanes of the source vector or vectors to use.
```